src/views/Recorder/Recorder.tsx

```diff
@@ -105,11 +105,8 @@ export function Recorder() {
       })
 
       const har = proxyDataToHar(grouped)
-      const prefix = startUrl && getHostNameFromURL(startUrl)
-      const fileName = await window.studio.har.saveFile(
-        JSON.stringify(har, null, 4),
-        prefix
-      )
+      const prefix = getHostNameFromURL(startUrl) ?? 'Recording'
```

Comment:
Here's another change: if no starting URL is provided, the name will follow the `Recording - YYYY-MM-DD` pattern